### PR TITLE
PUBD-1329: Add Klaro consent manager

### PR DESCRIPTION
**Pull request by hardyoyo**
Add Klaro Consent Manager, fixes PUBD-1329

### CODE REVIEW

**Comment by bibliotechy** [comment]
@hardyoyo Privacy statement to link to is https://cdlib.org/about/policies-and-guidelines/privacy-statement/

**Comment by hardyoyo** [comment]
Hmmm, OK, should we change the other link in the footer, too?

File: app/jsx/App.jsx [hunk 1]
```diff
@@ -13,6 +13,11 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import ReactGA from 'react-ga'
 import ReactModal from 'react-modal'
 
+// Import klaroConfig
+import klaroConfig from './klaro-config.jsx';
+// Log klaroConfig to the console
+console.log(klaroConfig);
```
Comment (everreau) [inline app/jsx/App.jsx:19]: Do you want to delete this?

File: app/jsx/App.jsx [hunk 2]
```diff
@@ -35,6 +40,26 @@ let prevPathname = null
...
+  componentDidMount() {
+    // Check if window is defined to ensure the Klaro code runs in a browser
+    // environment (not ISO/server side rendering)
+    if (typeof window !== 'undefined') {
+      // Dynamically import Klaro only in the browser environment
+      import('klaro').then((Klaro) => {
+        window.Klaro = Klaro;
+        // Ensure klaroConfig is defined before using it
+        if (typeof klaroConfig !== 'undefined' && klaroConfig !== null) {
+          console.log('klaroConfig is defined:', klaroConfig);
```
Comment (everreau) [inline app/jsx/App.jsx:53]: Maybe this one too? or maybe just a generic message "klaro loaded"

**Review by everreau — COMMENTED** [review]
Looks good to me.  Just the few console logs that might be overkill.

File: app/jsx/App.jsx (hunk 1 above)
Comment (hardyoyo) [reply app/jsx/App.jsx:19]: I kind of like leaving these in, as proof to Klaro skeptics.

Comment (hardyoyo) [reply app/jsx/App.jsx:19]: Oh, though... this is the entire klaroConfig... that's not necessary, I'll take that one out

**Comment by bibliotechy** [comment]
Closing so I can recreate it with a branch we own.

**Pull request closed without merging**